### ISSUE TITLE
Go to OSCON 2017

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#948]
So, funny story ... I approached Jono Bacon in private email asking for a favor, help with an invite to #930. But my email was so poorly worded that he thought I was _offering_ to help with the [Community Leadership Summit](http://www.jonobacon.org/2016/11/21/2017-community-leadership-events-update/) that he runs ahead of OSCON. So here we are! It's about time I went to OSCON, anyway. :-)

---------------------------

[CLS](http://www.communityleadershipsummit.com/): May 6-7
[OSCON](http://conferences.oreilly.com/oscon/oscon-tx): May 8-11

---------------------------

**Friday, May 5**
- Luke arrives at 5:?? PM to check in to AirBNB
- Chad arrives at 6:45 PM, Luke picks Chad up
- [CLS](http://www.communityleadershipsummit.com/schedule/) opener. [Bangers](https://www.bangersaustin.com/), 79 Rainey St, Austin, 78701 starting at 7pm

**Saturday, May 6**

- [CLS](http://www.communityleadershipsummit.com/schedule/) 8:30 AM to late

**Sunday, May 7**

- [CLS](http://www.communityleadershipsummit.com/schedule/) 9:00 AM to 6:00 PM

**Monday, May 8**

**Tuesday, May 9**

**Wednesday, May 10**

- [OSCON](https://conferences.oreilly.com/oscon/oscon-tx)
- sushi w/ Kapil (Capital One)
- \+ @calvinhp 

**Thursday, May 11**

- [OSCON](https://conferences.oreilly.com/oscon/oscon-tx)
- 9:30 [Crepes](http://www.cafecrepeofaustin.com/) w/ Salesforce crew
- \+ @calvinhp 

**Friday, May 12**

- Checkout
- Chad departs at 11:05 AM
- Luke departs at ?

**Comment by whit537** [comment]
| item | cost ($) |
|:-------|--------:|
| OSCON | 1,400 |
| room (7 nights, Airbnb) | 350 |
| flight | 300 |
| meals & entertainment | 350 |
| total | 2,400 |

**Comment by whit537** [comment]
https://www.google.com/search?q=oscon+free suggests that they've offered free "expo hall" passes in the past, which seem like they'd be a good fit for how I like to approach conferences anyway.

http://conferences.oreilly.com/oscon/oscon2011/public/content/free
http://conferences.oreilly.com/oscon/oscon2012/public/content/free-expo

Worth emailing them, I think. For $1,000 I think this would be a no-brainer.

**Comment by whit537** [comment]
I [called](http://conferences.oreilly.com/oscon/oscon-tx/public/content/contact) and talked to a rather unhelpful customer support rep. Advised to email. Sounds like they don't announce the expo hall registration until a week or two before the conference. Dropped a DM:

> Greeting! Will you be offering the expo hall registration again this year?

> I'm not seeing it on the website at https://conferences.oreilly.com/oscon/oscon-tx/public/register … but I do find it for many past years.

**Comment by whit537** [comment]
I can't imagine they won't offer the expo pass again. They've apparently offered it for the past eight years!

**Comment by whit537** [comment]
| item | cost ($) |   |
|:-------|--------:|---|
| OSCON | 0 | free with expo pass |
| room (7 nights, Airbnb) | 450 | walking distance |
| flight | 350 |  |
| meals & entertainment | 400 |  |
| total | 1,200 |  |

**Comment by nobodxbodon** [comment]
So is it a reasonable deadline of https://github.com/gratipay/gratipay.com/issues/4148? Or something else that can make a hit?

**Comment by whit537** [comment]
> So is it a reasonable deadline of gratipay/gratipay.com#4148?

Yes, I would hope to have [Integrate npm](https://github.com/orgs/gratipay/projects/8?fullscreen=true) done before OSCON.

**Comment by whit537** [comment]
@clone1018 If you go to OSCON what days/nights would you expect to be in Austin? I guess that's the first question, and then we can spec out a shared Airbnb.

**Comment by clone1018** [comment]
@whit537 I live 3 hours away so whatever is fine 🚗

**Comment by whit537** [comment]
DM response from OSCON:

> We usually do, but only announce it after best price and early price are over.

**Comment by whit537** [comment]
Pretty sure @calvinhp is willing to split a room with us, @clone1018.

**Comment by calvinhp** [comment]
I could be down with that

**Comment by whit537** [comment]
Flight booked! Arriving Friday, May 5, departing Friday, May 12.

**Comment by whit537** [comment]
@clone1018 @calvinhp What nights will you be in town?

**Comment by whit537** [comment]
I'm finding places for $1,500 within walking distance, and closer to $1,000 five miles out. That's two or three bedrooms.

**Comment by clone1018** [comment]
Sorry I've been MIA :P. I have no set plans right now, I can be there anytime and I'll be driving so we can go wherever. :car: 

Does that help with the housing cost @whit537 ?

**Comment by whit537** [comment]
It sounds like if @calvinhp makes it, it will only be for the conference itself, May 10-11, and he's willing to crash on a floor or whatever. Time to book an Airbnb ...

**Comment by whit537** [comment]
I've requested [this place](https://www.airbnb.com/rooms/4654381). It's 1½ from the convention center.

**Comment by whit537** [comment]
... and it sounds like @calvinhp is highly unlikely at this point, can't find a good flight. 😞

**Comment by clone1018** [comment]
I've received the AirBnb invite, are we locked in now?

**Comment by whit537** [comment]
We are! 💃

![austin](https://cloud.githubusercontent.com/assets/134455/24556748/646ae940-1603-11e7-9adc-91addd7bcd40.jpg)

**Comment by whit537** [comment]
I've asked if she's okay with a shout-out on Twitter.

> We are getting excited for @ oscon and #cls17! Just booked this cute cottage near the lake as home base for the week. ⛵️ 🚲 😀

**Comment by whit537** [comment]
I just registered for http://www.communityleadershipsummit.com/.

@clone1018 You planning to attend that, or just OSCON, or ... ?

**Comment by clone1018** [comment]
I'm on the ride for anything @whit537

**Comment by whit537** [comment]
Okay. My plan is to arrive Friday evening, get settled in, probably try to hit the CLS opener. Then participate in CLS on Saturday and Sunday. Monday and Tuesday for hacking, then OSCON on Wednesday and Thursday. Fly home Friday morning.

**Comment by whit537** [comment]
Pinged Jono to ask how I can help.

Heard back from host and shared [on Twitter](https://twitter.com/Gratipay/status/847892217452802049 ).

**Comment by dmk246** [comment]
That cottage looks so cute! Right next to a park with hiking trails!? :) Sounds great!  Hope you have a great time!

**Comment by whit537** [comment]
Jono has asked me to help with promoting CLS with local Austin people. He linked me to a spreadsheet with a couple hundred orgs listed, and I've started sending out 10-ish emails a day (from my gratipay.com address, btw). Yay, networking? :)

**Comment by whit537** [comment]
> 6 / 28 = 21% response rate :dancer:
> Giving me confidence for June when we’ll be ready to make a marketing push for Gratipay.

[slack](https://gratipay.slackarchive.io/gratipay/page-83/ts-1491410906114847)

**Comment by whit537** [comment]
Week one of outreach (funnel numbers are preliminary, might be a little higher if I dredge threads):

![](https://cloud.githubusercontent.com/assets/134455/24819632/6977d0be-1bb3-11e7-992b-de55eacf0187.png)

Waiting to hear back from Jono before investing too much more time in new sends.

**Comment by whit537** [comment]
Heard back. Still rollin' with this, though not as time-intensive as the first week.

**Comment by whit537** [comment]
Sounds like @calvinhp is going to join us after all! 💃

**Comment by whit537** [comment]
Expo pass is $99 this year:

https://conferences.oreilly.com/oscon/oscon-tx/public/content/exhibitplus

**Comment by calvinhp** [comment]
Expo Plus NG Pro Pass acquired!

**Comment by clone1018** [comment]
What do I need to do @whit537 ?

**Comment by whit537** [comment]
@clone1018 I guess buy a ticket? Should we ask Gratipay to spring for 'em?

**Comment by clone1018** [comment]
I don't mind paying for my ticket since I get personal growth out of it

**Comment by whit537** [comment]
I think Gratipay [paid](https://github.com/gratipay/inside.gratipay.com/issues/920#issuecomment-273922534) for our $ustain tickets? I think Gratipay could/should pay for tix here as well. Isn't that what we decided at some point? We pay for travel/room and Gratipay pays for events?

**Comment by clone1018** [comment]
Let's do it then

**Comment by whit537** [comment]
Registered!

**Comment by clone1018** [comment]
Registered also

**Comment by whit537** [comment]
@clone1018 Oh no! You didn't buy a ticket, did you? I bought two tickets, one for each of us. I should've said that. :(  If you bought your own can you get a refund?

**Comment by clone1018** [comment]
Err, I got an email from OSCON from you saying I needed to signup:

> You've been signed up by Chad Whitacre
> to attend O'Reilly Open Source Convention in Austin 2017.

> Please sign in or register for an O'Reilly account to take full
> advantage of attendee features:

**Comment by whit537** [comment]
Okay, awesome. Phew. :-)

**Comment by whit537** [comment]
Finished first pass on outreach for CLS. Quite the experience! I didn't have anybody bite my head off, and I found several people who were enthusiastic about the event. I am definitely inspired to do something similar to spread the word about Gratipay!

![screen shot 2017-04-27 at 8 41 37 am](https://cloud.githubusercontent.com/assets/134455/25484126/b5ed4bfe-2b26-11e7-9de7-c8f6e5724769.png)

**Comment by clone1018** [comment]
@whit537 I just realized this was next week! What day are you heading down, what airport should I pick you up from, etc

**Comment by whit537** [comment]
Wednesday night sushi w/ @calvinhp and @/kapilt!

**Comment by whit537** [comment]
@clone1018 I arrive at 6:45 PM at AUS. I've started an itinerary in the ticket description.

**Comment by whit537** [comment]
... this Friday, that is. What nights do you think you'll be down? We have the place from May 5 through 12. CLS is this weekend, and the main OSCON conference is a week from tomorrow.

**Comment by clone1018** [comment]
Alright I updated the ticket description with my plan details. I'll be there Friday 05 to Friday 12

**Comment by whit537** [comment]
I cannot tell a lie. 🙊

> Hi, Anna! Looking forward to discovering the Cottage this Friday!
>
>I wanted to follow up with you about the number of guests to make sure we're within expectations. It turns out that Luke would like to join me for the full week (not just a couple nights), and our other colleague Calvin is interested in staying with us for two nights next week (Wednesday and Thursday). I had originally thought one or the other might join for a couple nights, but the pictures of the Cottage must have convinced them both! :-)
>
>Is this within what you are comfortable with? Let me know! Thank you! :-)

**Comment by whit537** [comment]
Total cost was $689.00 for seven nights, or $98.43 per night.

We're splitting two nights three ways. Two nights is $196.86, split three ways is $65.62.

So that's $65.62 for @calvinhp.

That leaves $623.38 between @clone1018 and @whit537, or $311.69 each.

Is my math right?

**Comment by clone1018** [comment]
The listing says it accommodates 4, would we be able to find anyone else to split it one more way? I guess I never fully realized the cost of the AirBNB. If not let me see what I can make work

**Comment by whit537** [comment]
Chip in whatever you can, @clone1018. You're contributing a 🚗  anyway. :)

**Comment by calvinhp** [comment]
I'm happy to chip in more if needed.

**Comment by clone1018** [comment]
Cool, we can figure it out when we get there.

**Comment by clone1018** [comment]
@whit537 What's your favorite type of fruit for a Kolache?

**Comment by whit537** [comment]
Oh gosh that sounds awesome. I usually get the savory ones but fruit sounds awesome too. Follow your heart? 💃

**Comment by whit537** [comment]
We are confirmed for crepes with Salesforce on Thursday morning! 💃 🇫🇷

**Comment by whit537** [comment]
> Is this within what you are comfortable with?

tl;dr Yes. All systems go for the Cottage!

**Comment by whit537** [comment]
**Good first day!** Luke opened up the Cottage and then picked me up at the airport in his 1991 Mazda pickup truck(!). Turns out the kolaches are a specialty from [Czech Stop](http://www.czechstop.net/), which is half-way between Dallas and Austin—a real treat! We walked the river trail over to Bangers for the CLS opener, caught Jono and Hibbets first thing in the door, met some Joomla and Mattermost crew. Drinks were sponsored by Mattermost (Old Rasputin on nitro!), met the founder, pitched him on Gratipay, he was like, "The problem you're going to run into is that people are too polite. They'll tell you it's a good idea to support open source but then won't actually do it." It was stimulating and he's not wrong.

Tomorrow we have CLS and we also have an invitation to a birthday party for one of the co-founders of (Shark-Tanked, Cuban-funded) [Beatbox Beverages](https://beatboxbeverages.com/). Her parents are acquaintainces from back home in Ambridge(!) and I had lunch with them at the Pittsburgh airport(!!) though they were on some $50 flight through Allegiant(!!!) instead of the United(!!!!) flight I was on. ¯\\\_(ツ)\_/¯

Talked about https://github.com/gratipay/inside.gratipay.com/issues/987#issuecomment-299612474, etc., etc., etc.

On to the next!

**Comment by whit537** [comment]
**Good second day!** The folks here at CLS tend to be people at larger companies who are responsible for managing the company's relationship with its developer community. We did meet a couple of folks that are community organizers in the political sense ("We get a few every year"—Jono) and I shared GitHub Ambridge with them. 💃  I also introduced them to [Bijoy](https://en.wikipedia.org/wiki/Bijoy_Goswami), another local who is the one person I've met who came in via the ✉️  outreach. Jono did call me up front and recognize that work, which was cool and only slightly awkward. I connected [with Lisa](https://twitter.com/SWDevAngel/status/861033277137846274) later in the day partly through that. 🍸 

Back to the top &hellip; the keynotes were from people at Magento, OpenSource.com (so I was buried there in some of the stats ;), Intel (I learned about "[share of voice](https://www.google.com/search?q=share+of+voice)"), and All Things Open. We had lunch with [Yaw](http://www.anokwa.com/) and [Hélène](https://github.com/lognaturel) from [OpenDataKit](https://opendatakit.org/), Georg and Chris from Bosch, [Cecilia](https://blog.opentechstrategies.com/author/cdonnelly/) from Open Tech Strategies, and [Daniel](https://twitter.com/dizquierdo) from Bitergia. (An aside: the lake here is actually the Colorado River, damned at both ends &hellip; but not _the_ Colorado River. ;-). After lunch we participated in Hélène's session on code review along with Georg and Chris, Krislin from the Navy, and Joram from Mattermost. I think for us the session validated that Gratipay has a pretty solid dev process for our size &hellip; and a pretty awesome culture. ☺️

Mid-afternoon we slipped out for the Beatbox co-founder's birthday. It was nice! Gave her roses and chocolates and met the other founders and some other friends, including Damien from [Everything](https://en.wikipedia.org/wiki/Everything_(video_game)). And oh yeah we drank some Beatbox (@clone1018 has a picture of a picture 😁 ) and @clone1018 ate lots of Syrian food and I got a temporary tattoo.

We landed back at CLS in time for ☕️  with Jono and Steven Walli (whom I first met at ATO, #757) as well as Jeremy and Elon from [Linux Fund](http://www.linuxfund.org/) (had a good chat—like us, they've moved just over $1M and have $100k in escrow 😆 ). At some point in the day [Danese](https://en.wikipedia.org/wiki/Danese_Cooper) punched a penny and gave me the inner source pitch ("It's the long tail of open source because you're introducing people to open source in a less threatening environment so eventually they can contribute to real open source," more or less).

Back at the Cottage at the end of the day we workshopped https://github.com/gratipay/inside.gratipay.com/issues/987#issuecomment-299612474 until pumpkin time.

**Comment by clone1018** [comment]
![image](https://cloud.githubusercontent.com/assets/226638/25781107/5ab15f42-32fa-11e7-93c1-4ba1054e3f3a.png)

**Comment by clone1018** [comment]
Side note for tomorrow's post, me and @whit537 attended this session https://docs.google.com/document/d/1y0p2V_dkSIFvatEy-mQyoFzipdVwKpILSlt_JFKvpuQ/edit

**Comment by whit537** [comment]
That was for yesterday's post. :P

**Comment by whit537** [comment]
**Good third day!** We arrived in the middle of keynotes yesterday morning. [Sticker ROI](https://twitter.com/glasnt/status/860879347556679680) was the slide winner. We actually hiked back to the hacienda to hack over lunch. That's when I wrote up #1057. Cut it close so drove back over for a 3:30 session on funding in open source hosted by [Cat Allman](https://www.linkedin.com/in/catallman/), open-source purse-string holder at Google(!). There were about 35 people at the session. Not really much new news for us but a good chance to meet people, including Cat and someone from IBM and someone from Docker. Failed to follow-up properly with Cat so trying to track down her email now, will keep an eye out for her this week.

I apologized to a couple folks for #319, one of whom was [pjf](https://twitter.com/pjf), a key customer support collaborator under Gittipay 1.0. It was great to finally meet in person! Paul and I were in a session on swag. I learned some things about scaling up a swag program (e.g.: send to a local corporate office and let the company handle international shipping—might work for cookies? ;-). That was the last session of the day. After the CLS wrap-up we ended up punching pennies and talking about the Gratipocalypse (somehow?) and then ended up talking over dinner with Hélène and [Abby Cabunoc Mayes](http://acabunoc.github.io/) about #1057. Abby has been in open science at Mozilla so we talked about that for a bit as well, her position recently expanded to encompass mentorship programs in a number of other areas. A group of us went to see the [bats](http://www.batcon.org/index.php/our-work/regions/usa-canada/protect-mega-populations/cab-intro) and I finally got to meet Karl Fogel! He works with Cecilia.

When we got home we finished the Beatboxes and fell asleep.

**Comment by whit537** [comment]
**Good fourth, fifth, and sixth days!** Monday and Tuesday we mostly hacked together. Highlight was getting gratipay/gratipay.com#4305 out the door! Woo-hoo! 💃 Today we picked up @calvinhp from the airport and spent the day in the expo hall. Saw a bunch of people and punched some pennies. GitHub, Changelog, PayPal, Postgres, FreeBSD, Huawei, RIT, OSI, SFC, LF, etc.. Dined on sushi with Kapil from Capital One. Stored the penny puncher in the speaker's lounge last night, storing it in the closet at the Capital One booth tonight. Gently seeding the idea for #BackTheStack (#1057) and also plugging Sustain (#920). @calvinhp and @clone1018 are still out, I'm turning in early so I can work in the morning. Day seven tomorrow! 🌞

**Comment by clone1018** [comment]
Seventh day!

... was a lot of parking :). The day was a rush from the moment I looked at my phone at 7:30AM to realize @whit537 scheduled our breakfast with Salesforce for 8:30AM instead of 9:30AM, oops. We did have any phone numbers for them but thankfully they check their emails :). We got that sorted out and headed out to Forthright. Just in case you're in Austin and want to check them out, they're attached to the side of the Four Seasons near Brazos... NOT on the water like we thought :)

![image](https://cloud.githubusercontent.com/assets/226638/26026025/75aacf18-37b9-11e7-9fba-c5e166d503e0.png)

We finally found Forthright and had some coffee while @calvinhp interviewed Chad about his story. Lots of python. Salesforce showed up on the dot, we got some breakfast and chatted about open source inside of Salesforce. The subject moved to funding open source (as it does) and our BackTheStack idea was met mildly. Everyone ended up walking to the convention center (I had to go run payday). Later when I met up with Chad he mentioned I missed the magical moment at the end of the conversation. Somehow the topic of developers within a company choosing the projects that would receive funding immediately lit up everyone's brains. It makes sense, right from the beginning John Resig has been trying to [develop this feature for us](https://johnresig.com/blog/gittip-at-khan-academy/). @whit537 will hammer out more of this idea later in a separate issue. 

The middle of the day consisted of the hallway track. @calvinhp was interviewed by Microsoft. We chatted with [OpenCollective](https://opencollective.com/) & [Zulip](https://www.zulip.org/) at the Open Source Alley 

![image](https://cloud.githubusercontent.com/assets/226638/26026118/4decccd6-37bb-11e7-8e4f-cf591b91603e.png)

We ended the convention by helping Ed from [Open Sourcing Mental Illness](https://osmihelp.org) and the [Software Freedom Conservancy](https://sfconservancy.org/) pack their booths up. Learned that convention packing is very hectic. From the second we started packing the booth all the curtains came down and carpet was being rolled up.

We ended the night with some folks from GitHub at a cocktail place called "Half Step" and I talked a lot :cold_sweat:

**Comment by whit537** [comment]
> developers within a company choosing the projects that would receive funding immediately lit up everyone's brains

https://github.com/gratipay/gratipay.com/issues/1153

**Comment by techtonik** [comment]
>A group of us went to see the bats and I finally got to meet Karl Fogel! 
http://www.batcon.org/index.php/our-work/regions/usa-canada/protect-mega-populations/cab-intro
```
403: Access Forbidden
Your location (BY) has been blacklisted.
```
=/ I can't solve the captcha through web proxy. Can you tell the about the problem? We have bats too!

**Comment by mattbk** [comment]
It's a site about bat conservation...

> Bat Conservation International (BCI) has been instrumental in protecting and promoting the now famous Congress Avenue Bridge bat colony as an eco-tourism destination, but as our name indicates, we work worldwide to protect bats and their habitats. Despite the popularity of the bridge bats in Austin, bats are still among the world's least appreciated and most endangered animals. Like other wildlife, bats suffer from habitat loss and environmental pollution, and now, the added mortality from White-nose syndrome and wind turbines, but persecution from humans remains a primary cause of their decline.

**Comment by whit537** [comment]
We also had coffee with [a VC](http://generalcatalyst.com/team/donald-fischer/). Both he and the OC crew admired us for our persistence. ☺️

**Comment by whit537** [comment]
👊 👊 🍖
